mico-core/src/main/java/io/github/ust/mico/core/web/ServiceController.java

```diff
@@ -317,9 +318,9 @@
     }
 
     @PostMapping(PATH_GITHUB_ENDPOINT)
-    public ResponseEntity<Resource<MicoService>> importMicoServiceFromGitHub(@RequestBody CrawlingInformation crawlingInformation) {
-        String uri = crawlingInformation.getUri();
-        String version = crawlingInformation.getVersion();
+    public ResponseEntity<Resource<MicoService>> importMicoServiceFromGitHub(@Valid @RequestBody GitHubCrawlingInfoDTO crawlingInfo) {
+        String uri = crawlingInfo.getUrl();
```

Comment:
You changed the DTO property `uri` to `url` but not this temporary variable `uri`